### ISSUE TITLE
city subdivision

### ISSUE THREAD

**Issue by a-cinchetti** [openpolis/geojson-italy#13]
I'm looking for city subdivisions but I can't find them.
So, if there are those subdivisions, can you tell me please where to find them? And maybe how to read the json to get this information.
Thanks

**Comment by guglielmo** [comment]
If you are looking for the sudivisions of the municipal areas: https://github.com/openpolis/geojson-italy/blob/master/topojson/limits_IT_municipalities.topo.json (in topojson format), or https://github.com/openpolis/geojson-italy/blob/master/geojson/limits_IT_municipalities.geojson (in geojson format)

If you are looking for the metropolitan areas (big cities), then there is no such layer, but you can extact the areas from the provinces, https://github.com/openpolis/geojson-italy/blob/master/topojson/limits_IT_provinces.json, just find a list of all metropolitan areas somewhere.

**Comment by a-cinchetti** [comment]
I mean the districts or neighborhood of a big city. I don't know if it is possible

**Comment by guglielmo** [comment]
There is some data at ISTAT, as mentioned in #11, but it's not included in this project.
I keep this open as it may be useful to work on it sometime.

**Comment by a-cinchetti** [comment]
I got it. I've found a link from istat that has these data. 

- [https://www.istat.it/it/archivio/104317](url)
- [https://www.istat.it/storage/cartografia/basi_territoriali/aree-subcomunali-xls-shp/ASC2011_WGS84.zip](url) this is the zip file

![image](https://user-images.githubusercontent.com/26360033/142206705-534b5a5c-f141-4be2-9b75-0426e6f10381.png)

this is only for GeoJSON and I don't know how to transform it into TopoJSON, maybe you know how.

This file includes a lot of urban divisions in all Italy such as Milan or Turin. This pic is an example of north Italy.

![image](https://user-images.githubusercontent.com/26360033/142207290-12a0ae0e-d6a5-4ebf-9d3f-ea5141907535.png)

As you can see there are Milan, Monza, Bergamo...

![image](https://user-images.githubusercontent.com/26360033/142207390-7e874dca-913a-4c01-b212-5e8ebaa91815.png)

For Milan and Rome there are other divisions beyond this, but I think there are the best.
I think a map of all municipales with these divisions instead big city (for instance: Milan quartiers instead Milan itself) could be a better GeoJSON, like this. 
![image](https://user-images.githubusercontent.com/26360033/142207841-3538b588-0e01-4bf4-8075-bebdf6585971.png)

Hope this could be helpful.

**Comment by guglielmo** [comment]
Yes. The only problem with these data is that they're old (2011) And city subdivisions have changed (for example in Rome).

**Comment by a-cinchetti** [comment]
Can I ask you how you created the comuni.geojson file? because even if these subdivsions are old I need them to replace the big city. Thanks
